queue_services/business-emailer/src/business_emailer/email_processors/__init__.py

```diff
@@ -36,6 +36,7 @@ def get_filing_info(filing_id: str) -> tuple[Filing, dict, dict, str, str]:
     if filing.business_id:
         business = Business.find_by_internal_id(filing.business_id)
         business_json = business.json()
+        business_json["businessName"] = business.legal_name
```

Comment:
Its better to be more specific, so that we can track why we have this code
```suggestion
        if business.legal_type in ("SP", "GP"):
            business_json["businessName"] = business.legal_name  # `legalName` in json would be replaced for firms
```